scripts/dump-ratings-reading-log.sh

```diff
@@ -0,0 +1,25 @@
+psql -U openlibrary -d openlibrary -c "copy (
+  select
+  concat('OL', bookshelves_books.work_id, 'W') as work_id,
```

Comment:
I believe this is the format of the `Key` column in the OL data dumps; let's keep it the same.

```suggestion
  concat('/works/OL', bookshelves_books.work_id, 'W') as work_id,
```